woocommerce/woocommerce
# My subscription product updates

### Changes proposed in this Pull Request:

Adds support for updating themes and extensions from the My Subscriptions page.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Manually install a outdated product version
2. Purchase a license for this product on WCCOM
3. On the [new subscriptions page](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions&tab=my-subscriptions) see the update button and confirm that clicking on it doesn't do anything
4. Activate the product from the [old subscriptions page](http://localhost:8888/wp-admin/admin.php?page=wc-addons&section=helper)
5. On the [new subscriptions page](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions&tab=my-subscriptions)  click update and confirm that your product was updated



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Add update support to the My subscriptions page.

#### Comment 

</details>
